src/main/java/org/folio/linked/data/integration/kafka/consumer/DataImportEventHandler.java

```diff
@@ -17,17 +18,21 @@
 @RequiredArgsConstructor
 public class DataImportEventHandler {
 
-  private final Marc2BibframeMapper marc2BibframeMapper;
+  private final MarcBib2ldMapper marcBib2ldMapper;
+  private final MarcAuthority2ldMapper marcAuthority2ldMapper;
   private final ResourceService resourceService;
 
   public void handle(DataImportEvent event) {
     if (isNotEmpty(event.getMarcBib())) {
-      var marc4ldResource = marc2BibframeMapper.fromMarcJson(event.getMarcBib());
+      var marc4ldResource = marcBib2ldMapper.fromMarcJson(event.getMarcBib());
       var id = resourceService.createResource(marc4ldResource);
       log.info("DataImportEvent with id [{}] was saved as LD resource with id [{}]", event.getId(), id);
     } else if (isNotEmpty(event.getMarcAuthority())) {
-      // TODO - Process MARC authority record
-      log.info("Processing MARC Authority record with event ID [{}]", event.getId());
+      var authorityResources = marcAuthority2ldMapper.fromMarcJson(event.getMarcAuthority());
+      var ids = authorityResources.stream()
+        .map(resourceService::createResource)
+        .toList();
+      log.info("Processing MARC Authority record with event ID [{}], saved with ids: {}", event.getId(), ids);
```

Comment:
Do we know a chunk size for an event containing authority data?
What if it's 1000? I think we don't want to pollute logs with 1000 ids :) May be we will write down ids count instead?

Reply:
As I understand, it's 1 (or 2)
But in general, I agree. I just wanted to make a similar to the  previous one